deployment/docker/Caddyfile

```diff
@@ -1,14 +1,14 @@
-0.0.0.0:80 {
-    log {
-        output stdout
-    }
-    @api {
-        path /api*
-    }
-    reverse_proxy @api scrumlr-backend:8080
+0.0.0.0:80, localhost:80, 127.0.0.1:80 {
```

Comment:
was this done do be more verbose regarding the bindings because
`0.0.0.0:80`
AND
`:80`

would be enough to create the same listenings